src/Arcus.Messaging.Pumps.Abstractions/Resiliency/DefaultMessagePumpCircuitBreaker.cs

```diff
@@ -82,7 +85,10 @@ public MessagePumpCircuitState GetCircuitBreakerState(string jobId)
         /// <exception cref="InvalidOperationException">Thrown when not a single or more than one message pump could be found by the configured job ID.</exception>
         protected MessagePump GetRegisteredMessagePump(string jobId)
         {
-            Guard.NotNullOrWhitespace(jobId, nameof(jobId));
```

Comment:
Since there were still references to the `Guard` class, this means that there must still be a reference to the `Guard.NET` project somewhere.  I believe we should find that reference and remove it from the csproj file as well.

Reply:
It is a transient package reference from Arcus.Observability. Not something that we explicitly reference.

Reply:
Ah Ok.  Thanks!